index.bs

```diff
@@ -2197,7 +2204,7 @@ The [=remote end steps=] with |command parameters| are:
 
 1. Run implementation-specific steps to set the [=top-level traversable/system focus=] on the |context| if it is not focused.
 
-    Note: This does not change the [=focused area of the document=] except as required to by other specifications.
+    Note: This does not change the [=focused area of the document=] except as <span class=allow-2119>required</span> to by other specifications.
```

Comment:
We could s/required to/mandated/ or something, if the problem is using RFC2119 keywords in non-normative text.